conf/messages

```diff
@@ -1373,6 +1373,20 @@ tai.taxCollected.atSource.otherIncome.description=Tax on other income not collec
 tai.taxCollected.byPAYE.title=Tax collected through your tax code
 tai.taxCollected.byPAYE.description=Pay As You Earn (PAYE) does not include tax paid elsewhere. The tax you’ll pay through PAYE is £{0}.
 
+
+# tax code change
+taxCode.change.journey.preHeading= Your tax code change
+taxCode.change.whatHappensNext.title=What happens next
```

Comment:
There are already three instances of this text in the messages file. So I would say it's best to move all 4 (including yours) into under one key and refactor across. Same goes for Welsh. This is only a suggestion since I think having page agnostic keys helps reduce duplication and increase readability when it comes to using it in views, and spec files.

Reply:
Discussed and will leave duplicates for the time being